src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/CaseSummaryOverviewResponse.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import java.time.LocalDate
+
+data class CaseSummaryOverviewResponse(
+  val personDetails: PersonDetails?,
+  val offences: List<Offence>?
+)
+
+data class PersonDetails(
+  val name: String?,
+  @JsonFormat(pattern = "yyyy-MM-dd", shape = JsonFormat.Shape.STRING)
+  val dateOfBirth: LocalDate?,
+  val age: Int?,
+  val gender: String?,
+  val crn: String?
+)
+
+data class ProbationTeam(
+  val code: String?,
+  val label: String?
+)
+
+data class Offence(
+  val mainOffence: String?,
```

Comment:
what does mainOffence mean in the context of this? Is it a Yes/No or something else?

Reply:
My mistake, I corrected the type to Boolean.